src/colormaps.jl

```diff
@@ -54,8 +54,8 @@ function init_colormaps()
 
     copy!(LinearSegmentedColormap, colorsm."LinearSegmentedColormap")
 
-    copy!(cm_get_cmap, cm."get_cmap")
-    copy!(cm_register_cmap, cm."register_cmap")
+    copy!(cm_get_cmap, matplotlib.pyplot."get_cmap")
+    copy!(cm_register_cmap, matplotlib.colormaps."register")
```

Comment:
Better to check for the new functions first in case someone is using PyPlot.jl with an older version of matplotlb:
```suggestion
    copy!(cm_get_cmap, haskey(plt, "get_cmap") ? plt."get_cmap" : cm."get_cmap")
    copy!(cm_register_cmap, haskey(matplotlib.colormaps, "register") ? matplotlib.colormaps."register" : cm."register_cmap")
```